code-security/admin_guide/get-started/setup-administrator-access.adoc

```diff
@@ -1,4 +1,21 @@
 == Set Up Administrator Access for Code Security
 
 You need to enable administrative access for all the DevSecOps and Security teams who need to add code repositories or pipelines, create policies and review scan results on Prisma Cloud.
-To know more see https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/prisma-cloud-administrator-roles.html[ add Prisma Cloud Administrators] and https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/prisma-cloud-admin-permissions.html[role permissions]. You can also see https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/add-prisma-cloud-users.html[add administrative users].
\ No newline at end of file
+To know more see https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/prisma-cloud-administrator-roles.html[add Prisma Cloud Administrators] and https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/prisma-cloud-admin-permissions.html[role permissions]. You can also see https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/add-prisma-cloud-users.html[add administrative users].
+
+=== Add a Custom Permission Group
+
+Administrators can create a custom permission group for Code Security on the Prisma Cloud console. Using the parameters for permissions, you can limit or enhance the responsibilities of the users.
+
+* *Suppression Management*: You can enable user permissions to view, create, update and delete resources on *Repositories* (Settings > Repositories). These parameters enable suppression management for vulnerabilities found in resources. In addition, configuring view permission for all Code Security functions allows you to see the resource vulnerabilities to make informed decisions.
```

Comment:
how does suppression management is related here? it should give you repository management and view / read only access for all screens